build.gradle

```diff
@@ -27,3 +31,22 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+dokka() {
```

Comment:
This dokka block is what configures the Gradle task internal to the Dokka dependency. In the future I might expand this to have different tasks, one that documents only public API members, one that documents all code, etc